common/src/main/java/org/apache/uniffle/common/config/RssBaseConf.java

```diff
@@ -158,6 +158,12 @@ public class RssBaseConf extends RssConf {
       .defaultValue(true)
       .withDescription("The switch for jvm metrics verbose");
 
+  public static final ConfigOption<String> RSS_SERVER_IP = ConfigOptions
```

Comment:
Is this config option is only used for test? We already have https://github.com/apache/incubator-uniffle/commit/6937631876052425b8d808d26caf78c79b24536a, could we use this?

Reply:
I think use environment variable is not good in this case, because we need create mutil ShuffleServer in ut sometimes.